src/industry_cmd.cpp

```diff
@@ -1644,6 +1646,30 @@ static void AdvertiseIndustryOpening(const Industry *ind)
 	Game::NewEvent(new ScriptEventIndustryOpen(ind->index));
 }
 
+/**
+ * Populate an industry's list of nearby stations, and if it accepts any cargo, also
+ * add the industry to each station's nearby industry list.
+ * @param ind Industry
+ */
+static void FindStationsNearby(Industry *ind)
```

Comment:
The name of this function implies it returns something, maybe `PopulateNearbyStations` ?